src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.TagsContainKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tag contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findTag";
```

Comment:
Using camel case for commands looks quite odd. Maybe `findtag` or `find-tag` would work better instead.